dragon/console.rb

```diff
@@ -42,6 +42,10 @@ def initialize
       @animation_duration = 1.seconds
       @shown_at = -1
       load_history
+
+      @config_fname = 'console_config.txt'
```

Comment:
Think of `init.el` in Emacs. It's just a code file.

Instead of parsing a `console_config.txt`. Just expose the APIs that allow someone to configure their console.

```ruby
# console-config.rb
$gtk.console.define_key_map key: :enter, method: :eval
$gtk.console.define_key_map key: [:ctrl, :v], method: :getclipboard
```

Reply:
`$gtk.console.docs` can print information about how to perform the customizations.

Reply:
And if they want to customize their console, they can do:

```ruby
# main.rb
require "app/console_config.rb"

def tick args
end
```

Reply:
@amirrajan Okay I'll see what I can do :>
Or both ways may be acceptable, since one might prefer to write some *sick* ruby code and some might prefer some shabby badly parsed config pseudolang :>

Reply:
> some shabby badly parsed config pseudolang

Ruby makes a beautiful pseudo-language ^_^. For example:

```ruby
class Object
  def define_key_map key_map_lookup, key_to_map, method
  end
end
```

now the config can look like 

```ruby
define_key_map :console, :enter, :move_up
define_key_map :console, [:ctrl, :v], :paste_from_clipboard
```